dips/dip-10.md

```diff
@@ -13,47 +13,57 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
+* Persistent Identifiers: Currently there is a lack of persistent user identifiers in the Diem ecosystem. Diem ID is a persistent identifier from a user's perspective. From the perspective of the chain, Diem IDs do not exist.
 
-# User story
+# End-to-End Experience
 Below is an example of using Diem ID domain for transferring money from one user to another. 
 
-* Receiver shares their Diem ID with the sender (outside the scope of this document)
-* Sender enters receiver's Diem ID and other relevant information into their wallet application
-* Sender's wallet attempts to fetch the receiver's profile picture and full name along with a confirmation dialog
-* Sender confirms the identity of the recipient
-* Sender presses ‘send’ button
-* Receiver's account is credited the amount in the transaction
-* Receiver's wallet displays recent transactions and their senders
+## Prerequisite:
+* VASPs get approval from association (via some offline process) on domain name
+```
+avasp, 0xf72589b71ff4f8d139674a3f7369c69b
+bvasp, 0xc5ab123458df0003415689adbb47326d
+```
+
+## User Story: 
+* Bob wants to receive funds from Alice
+* Bob registers a Diem ID at VASP B: `bob@bvasp`
+* Alice's Diem ID is registered at VASP A: `alice@avasp`
+* Bob shares the Diem ID, `bob@bvasp` with Alice
+* Alice logs into VASP A, enters Bob’s identifier, an amount to pay, and submits payment
+* Alice’s VASP (VASP A) contacts Bob’s VASP’s (VASP B) off-chain API with the sender identifier `alice@avasp` and requests a reference_id, `rb1`
+* VASP A constructs a transaction with the specified amount and the reference_id `rb` and submits it to the Diem network
+* VASP B receives a transaction with metadata containing `rb`, deposits the amount in Bob's account, and attaches the relevant metadata (e.g., Alice’s Diem ID) to the internally stored transaction so Bob can confirm the transaction
+
 
-# Diem ID format
-Diem ID is a string in format `<user-id>$<diem-id-domain>`.
+# Diem ID Format
+Diem defines globally unique identifiers to facilitate transacting on the Diem Payment Network. As Diem stores no PII on-chain, these identifiers are exchanged during the off-chain process between two VASPs to identify a distinct source and destinations within their respective VASPs in order to produce a reference_id that can be stored on-chain. The format for the identifiers follows:
+`[user_identifier]@[vasp_domain_identifier]`
 
-Example: `andrey$novi`
+Example: `alice@avasp`
 
-* **user-id**, identifies an account within a VASP:
+* `user_identifier` is a reusable identifier that represents either the source or destination end-user of a payment transaction. It is unique to per user at VASP level. Specification:
   * Case insensitive
```

Comment:
If these identifiers are case insensitive then why don't we just enforce that they're lowercase?

Reply:
Hmm..... it doesn't change the internal representation... my gut feeling is we're going to want more expressive character-set that maps to what at least the email standard is.